tests/cases/compiler/argumentsSpreadRestIterables.tsx

```diff
@@ -0,0 +1,22 @@
+// @strict: true
+// @noEmit: true
+// @target: es5, esnext
```

Comment:
Any reason to be testing `es5` here?

Reply:
Basically to ensure you can't spread strings in es5 and that errors are expected to be reported for that. There is `IterationUse.AllowsStringInputFlag` that is used by for-of that allows strings to be iterated over in es5 for that purpose. I'm not sure why it's like that (and IIRC it's only allowed since es5 so it's not allowed in es3, I'm also not sure why that is. So overall, this just makes sure that this flag is not used in es5 context.